packages/utils/src/internals/sitemap.ts

```diff
@@ -79,52 +285,6 @@ class SitemapTxtParser extends Writable {
 export class Sitemap {
```

Comment:
This class has become pretty hollow. I'd welcome any opinions on how to make it useful. Or can we remove it?